pkgs/top-level/all-packages.nix

```diff
@@ -23357,10 +23357,7 @@ with pkgs;
 
   vsqlite = callPackage ../development/libraries/vsqlite { };
 
-  vte = callPackage ../development/libraries/vte {
-    # Needs GCC ≥10 but aarch64 defaults to GCC 9.
-    stdenv = clangStdenv;
-  };
+  vte = callPackage ../development/libraries/vte { };
```

Comment:
```suggestion
  vte = callPackage ../development/libraries/vte {
    # Needs GCC ≥10 but aarch64 defaults to GCC 9.
    stdenv = stdenv = if stdenv.isLinux && stdenv.isAarch64 && stdenv.cc.isGNU then clangStdenv else stdenv;
  };
```

Reply:
Is it ok to squash this change with the `vte` commit and mark you as co-author?

Reply:
I think that needs to be squashed. I don't mind being a co-author or not, though.

Reply:
Squashed and co-authored-by with a slight format tweak